data-plane/docker/generate_jdk.sh

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env sh
```

Comment:
Since we use bash everywhere can we use it here as well? It's for our own sanity :smile:

Reply:
I used sh because i was initially trying with alpine, that has only sh and not bash. I prefer to keep as is, so on the next iteration i can work on using alpine more than ubuntu

Reply:
Ok, thanks!